schema/codelists/tenderStatus.csv

```diff
@@ -1,8 +1,8 @@
-Code,Title,Description
-planning,Planning,"A future contracting process is being considered. Early information about the process can be provided in the tender section. A process with this status might provide information on early engagement or consultation opportunities, during which the details of a subsequent tender can be shaped."
-planned,Planned,"A contracting process is scheduled, but is not yet taking place. Details of the anticipated dates can be provided in the tender block."
-active,Active,A tender process is currently taking place.
-cancelled,Cancelled,The tender process has been cancelled.
-unsuccessful,Unsuccessful,The tender process was unsuccessful.
-complete,Complete,The tender process is complete.
-withdrawn,Withdrawn,No further information on this process is available under this ocid.
+Code,Title,Description,Deprecated,Deprecation note
+planning,Planning,"A future contracting process is being considered. Early information about the process can be provided in the tender section. A process with this status might provide information on early engagement or consultation opportunities, during which the details of a subsequent tender can be shaped.",1.2,
+planned,Planned,"The contracting process is scheduled, but is not yet taking place. Details of the anticipated dates can be provided in the tender block.",,
+active,Active,The procurement documents have been published and the bid submission deadline has not yet passed.,,
+complete,Complete,The bid submission deadline has passed.,,
+cancelled,Cancelled,"The contracting process has been cancelled by the buyer or the procuring entity (e.g. because of a change in needs, insufficient funds, or technical or procedural errors) after the procurement documents have been published, but not later than the bid submission deadline.",,
+unsuccessful,Unsuccessful,"The contracting process failed after the procurement documents have been published, but not later than the bid submission deadline (e.g. no bids were received, all bids were withdrawn by the bidders, all bids were rejected by the buyer or the procuring entity).",,
```

Comment:
Unlike all the others, here the parenthetical is at the end. Can we move it, for easier comparison by a reader looking to distinguish the two options?

Reply:
Done in https://github.com/open-contracting/standard/pull/1509/commits/8aee47456cc68f8907f997c7801245201f29b81e. (The sentence before the parentheses is "The contracting process failed." is extremely broad, which is why I originally put the parentheses at the end, but you have large breadth also in the other stages, so it probably isn't a good reason.)